Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -55,8 +58,21 @@ public struct PreplannedListItemView: View {
     @ViewBuilder private var downloadButton: some View {
         switch model.status {
         case .downloaded:
-            Image(systemName: "checkmark.circle")
-                .foregroundStyle(.secondary)
+            Button {
+                Task {
+                    if let map = await model.loadMobileMapPackage() {
+                        onMapSelectionChanged?(map)
+                    }
+                }
+            } label: {
+                Text("Open")
+                    .font(.system(.headline, weight: .bold))
+                    .foregroundColor(.accentColor)
+                    .frame(width: 76, height: 32)
+                    .background(Color(UIColor.tertiarySystemFill))
```

Comment:
It can be `.background(Color(.tertiarySystemFill))`, but if we use the `backgroundStyle` API, then it can be 
```suggestion
                    .backgroundStyle(.tertiary)
```

Reply:
`.backgroundStyle(.tertiary)` does not appear to be equivalent to `.background(Color(.tertiarySystemFill))` since the background color does not show, so I used `.background(Color(.tertiarySystemFill))`.